packages/firestore/lib/index.d.ts

```diff
@@ -1349,7 +1358,7 @@ export namespace FirebaseFirestoreTypes {
      * Changes the behavior of `set()` calls to only replace the specified field paths.
      * Any field path that is not specified is ignored and remains untouched.
      */
-    mergeFields?: (string | FieldPath)[];
+    mergeFields?: string | FieldPath[];
```

Comment:
Shouldn't this still be an array of either string or field path, this is no longer an array after this change

Reply:
I think that was prettier actually, didn't manually change it. I guess correct types are `string[] | FieldPath[]`